src/panels/config/automation/trace/ha-automation-trace.ts

```diff
@@ -254,6 +285,16 @@ export class HaAutomationTrace extends LitElement {
           top: 8px;
           right: 8px;
         }
+
+        .details {
+          display: flex;
```

Comment:
So it is not sticked to the sides
```suggestion
          display: flex;
          margin: 0 16px;
```